lib/ttt.ex

```diff
@@ -22,6 +22,7 @@ defmodule TTT do
 
   def get_current_move(is_valid, game) do
     elem(is_valid, 1)
```

Comment:
Maybe rather than doing `elem` you could make it more readable with:
```
def get_current_move({valid_move, _}, game) do
    valid_move
    |> display_move(game.current_player.marker)
```